src/installing-rust.md

```diff
@@ -15,18 +15,25 @@ $ rustup update nightly
 ```
 
 This installs the current version of nightly Rust. We run all of the examples
-in this book under continunous integration, so we should know if something
+in this book under continuous integration, so we should know if something
 changes in nightly Rust and breaks. But please [file bugs] if something doesn't
 work.
 
 [file bugs]: https://github.com/intermezzOS/book/issues/new
 
-Then, execute this:
+You shouldn't use nightly Rust unless you need to, which is why `rustup` allows
+you to override the default version whenever you're in a particular directory.
+We don't have a project directory yet, so let's create one:
```

Comment:
So, we do this over at http://intermezzos.github.io/book/multiboot-headers.html#writing-our-own-multiboot-header

I can see why doing it here is also nicer, but we should remove that as well.